src/plugins/global-settings/global-colors/index.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * Internal dependencies
+ */
+import { GlobalSettingsPanel, GlobalSettingsColorPicker } from '../components'
+
+/**
+ * External dependencies
+ */
+import { i18n } from 'stackable'
+import md5 from 'md5'
+
+/**
+ * Wordpress dependencies
+ */
+import {
+	addFilter, addAction, doAction,
+} from '@wordpress/hooks'
+import { Fragment } from '@wordpress/element'
+import { __ } from '@wordpress/i18n'
+import domReady from '@wordpress/dom-ready'
+import { loadPromise, models } from '@wordpress/api'
+import { dispatch, select } from '@wordpress/data'
+
+addFilter( 'stackable.global-settings.inspector', 'global-settings/global-colors', output => {
+	return (
+		<Fragment>
+			{ output }
+			<GlobalSettingsPanel
```

Comment:
Please use `PanelAdvancedSettings` instead. Check for existing components first before creating one. We have a lot of components and utility functions.